api/app/Notifications/EventCreatedNotification.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Entity\Event;
+use App\Entity\EventType;
+use App\Entity\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class EventCreatedNotification extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    private Event $event;
+    private EventType $eventType;
+    private User $user;
+
+    /**
+     * Create a new notifications instance.
+     *
+     */
+    public function __construct(Event $event)
+    {
+        $this->event = $event;
+        $this->eventType = $event->eventType;
+        $this->user = $event->eventType->owner;
+    }
+
+    /**
+     * Get the notifications's delivery channels.
+     *
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notifications.
+     *
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage())
+            ->subject("New Event: {$this->event->invitee_name},
+                    {$this->event->start_date}, {$this->eventType->name}")
+            ->markdown('notifications.event_created', [
+                'event' => $this->event,
+                'eventType' => $this->eventType,
+                'user' => $this->user
+            ]);
```

Comment:
All this content should not be here. Please, create [Mailable](https://laravel.com/docs/7.x/mail#generating-mailables) class that will contain all these things.